packages/tags/src/views/Close.js

```diff
@@ -0,0 +1,37 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
+import classNames from 'classnames';
+import TagStyles from '@zendeskgarden/css-tags';
+import { retrieveTheme } from '@zendeskgarden/react-theming';
+
+import { version } from '../../package.json';
+const COMPONENT_ID = 'tags.close';
+
+/**
+ * Used to close a Tag. Supports all `<div>` props.
+ */
+const Close = styled.div.attrs({
+  'data-garden-id': COMPONENT_ID,
+  'data-garden-version': version,
+  'aria-label': 'Press delete to remove',
```

Comment:
Do we typically inject a default (english) message for ARIA?

Reply:
Normally this is abstracted into a container component, but since this is the only property and there isn't any other interaction I think that would be a little overkill.

We default to english in the original `Label` component, but to help make this more visible I've added in a large `ACCESSIBILITY WARNING` message to inform people that this property will need to be localized.